paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/PaymentSheetPlaygroundActivity.kt

```diff
@@ -161,33 +163,35 @@ internal class PaymentSheetPlaygroundActivity : AppCompatActivity() {
     }
 
     @Composable
-    private fun ReloadButton(
+    private fun ReloadSection(
         playgroundSettings: PlaygroundSettings,
         isReloading: Boolean,
     ) {
-        Button(
-            onClick = {
-                viewModel.prepareCheckout(
-                    playgroundSettings = playgroundSettings,
-                )
-            },
-            enabled = !isReloading,
-            modifier = Modifier
-                .fillMaxWidth()
-                .testTag(RELOAD_TEST_TAG),
-        ) {
-            if (isReloading) {
-                CircularProgressIndicator(
-                    strokeWidth = 1.dp,
-                    modifier = Modifier
-                        .requiredSize(16.dp)
-                        .padding(end = 16.dp),
-                )
+        val autoRefresh by playgroundSettings[AutoRefreshSettingsDefinition].collectAsState()
 
-                Text("Reloading…")
-            } else {
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            modifier = Modifier.fillMaxWidth(),
+        ) {
+            Button(
+                onClick = { viewModel.prepareCheckout(playgroundSettings) },
+                enabled = !isReloading && !autoRefresh,
+                modifier = Modifier
+                    .weight(1f)
+                    .padding(end = 16.dp)
+                    .testTag(RELOAD_TEST_TAG),
+            ) {
                 Text("Reload")
             }
+
+            Text(text = "Auto")
+
+            Switch(
```

Comment:
Shouldn't this be just another setting? We shouldn't need a special case here for it.

Reply:
Ping